RELEASING.md

```diff
@@ -2,37 +2,48 @@
-   1. `Choose a tag` and create a tag for the new version. ex: `v1.0.0`
-   1. `Generate release notes`
+   1. `Choose a tag` and create a tag for the new version. ex: `v1.0.0`.
+   1. `Generate release notes`.
    1. Ensure the `Title` corresponds to the version we're publishing and the generated `Release Notes` are accurate.
    1. Hit `Publish release`.
 
-1. Publish to CocoaPods:
+At this point, the new release is available and can be consumed using the Swift Package Manager.
+
+[Square specific] To make the new version available internally, [bump the version](https://go/spm-bump-dependency) through SPM.
+
+### Publishing to CocoaPods
+
+1. You need to have 2 older Xcode versions, but you will only use them through the command line, so do not worry about the app being incompatible with your macOS version. You will also need to tweak one of them:
+   1. Download **Xcode 14.2**. This will be used to publish all pods except for `WorkflowRxSwift` and `WorkflowRxSwiftTesting`.
+   1. Download **Xcode 15.4**. This will be used to publish `WorkflowRxSwift` and `WorkflowRxSwiftTesting`.
+   1. Copy `Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/arc`from within the Xcode 14.2 app to the same location in Xcode 15.4.
+
+1. Make sure you are set up with 
+[`bundle exec pod trunk`](https://guides.cocoapods.org/making/getting-setup-with-trunk.html) and your CocoaPods
+account is a contributor for all pods. If you need to be added as a
+contributor, please [open a ticket requesting access](https://github.com/square/workflow-swift/issues/new).
+    1. For Squares, membership is managed through the `workflow-swift-owners` registry group. Please request access to that group through Registry.
+    1. Once you have access, you can register a session with `bundle exec pod trunk` using the group e-mail alias `workflow-swift-owners@squareup.com`:  
+`bundle exec pod trunk register workflow-swift-owners@squareup.com 'Workflow Swift Owners' --description='Your computer description'`.
+    1. You should receive an email from CocoaPods to confirm your session before you procede.
+
+1. To avoid possible headaches when publishing podspecs, validation can be performed first.
+    1. Switch to Xcode 15.4: `sudo xcode-select -s Path/to/Xcode15.4`.
+    1. Run `bundle exec pod lib lint WorkflowRxSwift.podspec WorkflowRxSwiftTesting.podspec`.
+    1. Ensure you're on Xcode 14.2: `sudo xcode-select -s Path/to/Xcode14.2`.
+    1. Run the following:
+    ```bash
+    bundle exec pod lib lint Workflow.podspec ViewEnvironment.podspec ViewEnvironmentUI.podspec WorkflowTesting.podspec WorkflowReactiveSwift.podspec WorkflowUI.podspec WorkflowReactiveSwiftTesting.podspec WorkflowSwiftUIExperimental.podspec WorkflowCombine.podspec WorkflowCombineTesting.podspec WorkflowConcurrency.podspec WorkflowConcurrencyTesting.podspec
+    ```
+
+1. Once validation passes on all pods, you can publish to CocoaPods:
```

Comment:
i thought the Xcode dance had to be done during the actual `pod trunk push` calls for the RxSwift dependencies

Reply:
That's right, I missed clarifying that this part also needs to be done with the 2 Xcode versions